cli/tests/pcluster/validators/test_iam_validators.py

```diff
@@ -113,3 +115,125 @@ def test_additional_iam_policy_validator(mocker, policy_arn, expected_get_policy
     mocker.patch("pcluster.aws.iam.IamClient.get_policy", side_effect=expected_get_policy_side_effect)
     actual_failures = AdditionalIamPolicyValidator().execute(policy=policy_arn)
     assert_failure_messages(actual_failures, expected_message)
+
+
+@pytest.mark.parametrize(
+    "resource_prefix, expected_message,expected_failure_level",
```

Comment:
We always expect the failure level to be `FailureLevel.ERROR` if a failure occurs. 
so, we can avoid the parameter `expected_failure_level` and assert `FailureLevel.ERROR` in the logic directly.

Reply:
Changing this string would lead me to make changes on the Function level as well, which would give an unresolved error as it wont compare with the parameter we are giving but the defined Enum we have.

Reply:
After a call, we have resolved to keep the test function as it is.